tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -232,6 +235,38 @@ Object.Pipeline {
 			}
 		}
 	}
+
+	passthrough-capture.2 {
+		index	7
+
+		Object.Widget.copier.1 {
+			stream_name	'Echo reference'
+		}
+	}
+
+	passthrough-copier-be.2 {
+		index	8
+
+		Object.Widget.copier."1" {
+			dai_index	1
+			dai_type	"SSP"
+			type		dai_out
+			copier_type	"SSP"
+			stream_name	"SSP1-Codec"
+			node_type	$I2S_LINK_INPUT_CLASS
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+			}
+		}
+
+		Object.Widget.copier."2" {
+			stream_name	"SSP0-Codec"
```

Comment:
why do you need to set this?

Reply:
Here I just followed the example of similar place like:
https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/cavs-nocodec.conf#L289
It's more out of convention than necessity.

Reply:
lets not set it blindly if it is not needed,